views/singlePost.pug

```diff
@@ -0,0 +1,6 @@
+extends layout
+
+block content
+  h2= title
+  .content.mce
```

Comment:
Do we need the `.mce` class here if we are just rendering the content and not editing it in the RTE?